### DISCUSSION TITLE
第1节课笔记(3班)

### DISCUSSION THREAD

**Discussion by matrixgame2018** [InternLM/Tutorial#53] [笔记]
第1节课笔记(3班)

**Comment by hydexi0122** [comment]
Hydexi-https://zhuanlan.zhihu.com/p/675927006

**Comment by SchweitzerGAO** [comment]
https://schweitzergao.github.io/2024/01/03/mmtutorial-1-note/

**Comment by EinesTages** [comment]
https://github.com/EinesTages/InternLM_Camp/blob/main/Lecture01/note.md

**Comment by SuperD0122** [comment]
http://t.csdnimg.cn/bU6Hh

**Comment by GoldWaterFall** [comment]
https://www.yuque.com/woshijinshui/wwi5m1/hqlqtce71vfbige4?singleDoc# 《书生·浦语大模型全链路开源体系》

**Comment by xiexiaoshinick** [comment]
https://zhuanlan.zhihu.com/p/676098903《书生.浦语大模型全链路开源体系》

**Comment by GeLink9999** [comment]
https://zhuanlan.zhihu.com/p/676153645

**Comment by strawberrylazy** [comment]
https://github.com/strawberrylazy/InterLM_Lesson/blob/main/lesson_1/%E4%B9%A6%E7%94%9F%C2%B7%E6%B5%A6%E8%AF%AD%E5%A4%A7%E6%A8%A1%E5%9E%8B%E5%AE%9E%E6%88%98%E8%AF%BE%EF%BC%88%E7%AC%AC%E4%B8%80%E8%8A%82%EF%BC%89.md

**Comment by zzmbhm** [comment]
https://z07ck55wvo.feishu.cn/docx/LjqkdMeJ8oBW3yxsKI0cq9ADnie

**Comment by wm901115nwpu** [comment]
https://zhuanlan.zhihu.com/p/676150746

**Comment by Sampson425** [comment]
第一节https://blog.csdn.net/m0_49908454/article/details/135403403

**Comment by 443752009** [comment]
http://t.csdnimg.cn/PtJaH

**Comment by semple030228** [comment]
https://blog.csdn.net/m0_49289284/article/details/135377601

**Comment by semple030228** [comment]
https://blog.csdn.net/m0_49289284/article/details/135377601

**Comment by GoooHi** [comment]
https://zhuanlan.zhihu.com/p/676172753

**Comment by Gty-2002** [comment]
https://npumgk6rgc.feishu.cn/docx/OcLFd4IJRoKAquxanoIcRyIhnyd?from=from_copylink

**Comment by Tallisgo** [comment]
http://t.csdnimg.cn/qX1z2

**Comment by fanqiNO1** [comment]
https://github.com/fanqiNO1/InternLM-tutorial/blob/main/section1/notes.md

**Comment by manmanmmm216** [comment]
[第一课笔记.docx](https://github.com/InternLM/tutorial/files/13928936/default.docx)

**Comment by Zhihao-z** [comment]
https://blog.csdn.net/m0_46870112/article/details/135588537?spm=1001.2014.3001.5501
